packages/strapi-design-system/src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -177,17 +177,19 @@ const GlobalStyle = createGlobalStyle`
   }
 `;
 
-export const ThemeProvider = ({ children, theme }) => {
+interface ThemeProviderProps {
+  children: React.ReactNode;
+  theme: DefaultTheme;
+}
+
+export const ThemeProvider = ({ children, theme }: ThemeProviderProps) => {
   return (
     <StyledThemeProvider theme={theme}>
-      <GlobalStyle />
-      {children}
-      <LiveRegions />
+      <>
```

Comment:
I'm wondering why would we need this empty tags? 🤔

Reply:
Good catch, i think originally it was erroring but that could have been a multitude of TS issues, removed now.